recipes/categorical_anomaly.py

```diff
@@ -269,15 +342,23 @@ def graph_scatter(self, **kwargs):
 
 
     def graph_geo(self, **kwargs):
+        """
+        If a uses provides a geodataframe with applicable geometries, this function maps those using Folium.
+
+        """
         if type(self.metrics_column) is str:
-            hover = [self.metrics_column]
+            hover = [self.metrics_column].copy()
```

Comment:
copy shouldn't be necessary as a new list is created anyway

Reply:
Ah, yup, sorry, this bug took me a while to figure out, I noticed sometimes metric column would have 'id' and 'anomaly' in it, figured out it came after calling graph_geo because I was appending to a variable I was pointing at self.metrics_column, figured I'd drop .copy() on both to be safe. I'll remove the top one.